### PR TITLE
Glob 72600/migrate to drone

### PR DESCRIPTION
**Pull request by marcglobality**
Related:
- https://github.com/globality-corp/globality-build/pull/690
- https://github.com/globality-corp/terraform-github/pull/512

Closes [GLOB-72600]

[GLOB-72600]: https://globality.atlassian.net/browse/GLOB-72600?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by salmanmashayekh — APPROVED** [review]
LGTM

File: globality_black/tests/fixtures/blank_lines_diff.txt [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
  def foo(df: pd.DataFrame, graph):
- 
+-
      df = (
          df
-@@ -27,27 +28,16 @@
+ 
+         # turn predictions to bool
+         .assign(predictions=df.predictions.astype(bool))
+@@ -27,27 +27,16 @@
```
Comment (tomaszpietruszka-globality) [inline globality_black/tests/fixtures/blank_lines_diff.txt:12]: Why the changes?

**Review by tomaszpietruszka-globality — APPROVED** [review]
Approving provided we get the go-ahead to migrate open-source repos to Drone

File: globality_black/tests/fixtures/blank_lines_diff.txt (hunk 1 above)
Comment (marcglobality) [reply globality_black/tests/fixtures/blank_lines_diff.txt:12]: a new version of black makes
```
class Foo

    def __init__(...):
        ....
```
into
```
class Foo
    def __init__(...):
        ....
```
so all tests started failing
